docs/book/src/topics/externally-managed-aws-infrastructure.md

```diff
@@ -1,8 +1,22 @@
-# Consuming Existing AWS Infrastructure
+# Externally managed AWS Infrastructure
 
-Normally, Cluster API will create infrastructure on AWS when standing up a new workload cluster. However, it is possible to have Cluster API re-use existing AWS infrastructure instead of creating its own infrastructure. Follow the instructions below to configure Cluster API to consume existing AWS infrastructure.
+Normally, Cluster API will create infrastructure on AWS when standing up a new workload cluster. However, it is possible to have Cluster API re-use external AWS infrastructure instead of creating its own infrastructure. 
 
-## Prerequisites
+There are two possible ways to make use of externally managed AWS infrastructure:
```

Comment:
It will be good to clarify what we mean here. We are not talking about EKS-managed (managed) clusters in the document. Because one might think EKS-managed clusters are also externally managed, their control-plane is managed by EKS.

Reply:
Done